### PR TITLE
Format doc on several modules

### PR DESCRIPTION
**Pull request by fertapric**

### CODE REVIEW

File: lib/phoenix/code_reloader/server.ex [hunk 1]
```diff
@@ -183,6 +183,7 @@ defmodule Phoenix.CodeReloader.Server do
         You must restart your server after changing the following config or lib files:
 
           * #{Enum.map_join(files, "\n  * ", &Path.relative_to_cwd/1)}
+
```
Comment (fertapric) [inline lib/phoenix/code_reloader/server.ex]: Depends on https://github.com/phoenixframework/phoenix/pull/3004#discussion_r209455704

File: lib/phoenix/token.ex [hunk 2]
```diff
@@ -205,6 +206,7 @@ defmodule Phoenix.Token do
 
         config :my_app, MyApp.Endpoint,
             secret_key_base: ...
+
```
Comment (fertapric) [inline lib/phoenix/token.ex]: Also depends on https://github.com/phoenixframework/phoenix/pull/3004#discussion_r209455704

File: lib/phoenix.ex [hunk 3]
```diff
@@ -86,6 +87,7 @@ defmodule Phoenix do
     configure an existing encoder in your mix config using:
 
         config :phoenix, :json_library, MyJSONLibrary
+
```
Comment (fertapric) [inline lib/phoenix.ex]: Also depends on https://github.com/phoenixframework/phoenix/pull/3004#discussion_r209455704

**Comment by jeregrine** [comment]
Thank you for all this. 💟

**Pull request merged**